sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -158,7 +186,7 @@ public static SqlFunction createZetaSqlFunction(String name, SqlTypeName returnT
         SqlFunctionCategory.USER_DEFINED_FUNCTION);
   }
 
-  private static SqlUserDefinedAggFunction createUdafOperator(
+  public static SqlUserDefinedAggFunction createUdafOperator(
```

Comment:
This (and `createUdfOperator`) is intended to be a (slightly hacky) helper function for implementing ZetaSQL operators via the UDF interface. Why do you need to make them public?

Reply:
In the original PR, I used createUdfOperator in ExpressionConverter, and createUdafOperator in AggregateScanConverter. So they can be package-private instead of public.